db/migrate/20200409155443_add_inet_partner_to_practice_partners.rb

```diff
@@ -0,0 +1,24 @@
+class AddInetPartnerToPracticePartners < ActiveRecord::Migration[5.2]
+  def change
+    iNet = PracticePartner.find_by(slug: 'vha-innovators-network')
+    unless iNet
+      PracticePartner.create!({name: 'VHA Innovators Network', short_name: 'iNET', description: 'The VHA Innovators Network (iNET) is a network of 33 VA medical centers changing the way employees think and solve problems through training and accelerated operationalizing innovation.', slug: 'vha-innovators-network'})
```

Comment:
Running `rake db:migrate` was throwing errors for me:
```
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

undefined method `id' for nil:NilClass
/Users/elisa/skylight/diffusion-marketplace/db/migrate/20200409155443_add_inet_partner_to_practice_partners.rb:21:in `block in change'
/Users/elisa/skylight/diffusion-marketplace/db/migrate/20200409155443_add_inet_partner_to_practice_partners.rb:17:in `each'
/Users/elisa/skylight/diffusion-marketplace/db/migrate/20200409155443_add_inet_partner_to_practice_partners.rb:17:in `change'

Caused by:
NoMethodError: undefined method `id' for nil:NilClass
/Users/elisa/skylight/diffusion-marketplace/db/migrate/20200409155443_add_inet_partner_to_practice_partners.rb:21:in `block in change'
/Users/elisa/skylight/diffusion-marketplace/db/migrate/20200409155443_add_inet_partner_to_practice_partners.rb:17:in `each'
/Users/elisa/skylight/diffusion-marketplace/db/migrate/20200409155443_add_inet_partner_to_practice_partners.rb:17:in `change'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```
setting `iNet = PracticePartner.create....` helped fix this issue for me...

However, spoke with @JoshingYou1 and Josh did not run into issue even though he also did not have a iNet PracticePartner in his DB... 🧐

Reply:
yes,
we need an `iNet =` on line 5 here.
thanks for catching that @emyl3.

Reply:
Updated! Thank you!